### ISSUE TITLE
Pulmo: Diffuse Lymphoid Hyperplasia

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#9755]
**Preferred term label:**
Diffuse Lymphoid Hyperplasia

**Synonyms**
DLH

**Definition (free text, please give PubMed ID)**
The term diffuse lymphoid hyperplasia (DLH) describes the pathological entity that lies between follicular bronchiolitis
and lymphocytic interstitial pneumonia (LIP). It is pathologically characterized by a diffuse peribronchiolar proliferation of hyperplastic lymphoid follicles (BALT), extended along the lymphatics to involve the interlobular septa and pleura. Unlike LIP, DHL does not show uniform involvement of the alveolar septa [19].

Citation: Review, Lymphocytic Disorders of the Chest: Pathology and Imaging. M. GIBSON and D. M. HANSELL. Clinical Radiology (1998) 53, 469-480

**Parent term (use hpo.jax.org/app)**
Pulmonary interstitial lymphocyte infiltration HP:0033582

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU chrapp, mgriese

**Comment by pnrobinson** [comment]
@GrieseAG @mgriese
WOuld it be possible to call this
Diffuse pulmonary lymphoid hyperplasia
so that the term is unambiguous outside of the hierarchy?
Could you please use ORCID ids -- we have switched the attribution pipeline!

**Comment by mgriese** [comment]
Yes, could add pulmonary, 
Christina K Rapp: ORCID ID: 0000-0002-7371-8158
Matthias Griese: ORCID ID: 0000-0003-0113-912X

**Comment by pnrobinson** [comment]
Thanks!
added as HP:0034824
